include/depends.mk

```diff
@@ -11,7 +11,7 @@
 
 DEP_FINDPARAMS := -x "*/.svn*" -x ".*" -x "*:*" -x "*\!*" -x "* *" -x "*\\\#*" -x "*/.*_check" -x "*/.*.swp" -x "*/.pkgdir*"
 
-find_md5=find $(wildcard $(1)) -type f $(patsubst -x,-and -not -path,$(DEP_FINDPARAMS) $(2)) -printf "%p%T@\n" | sort | $(MKHASH) md5
+find_md5=find $(wildcard $(1)) -type f $(patsubst -x,-and -not -path,$(DEP_FINDPARAMS) $(2)) -printf "%p%T@\n" | sort | awk -v srch="$(TOPDIR)" -v repl="" '{ sub(srch,repl,out); print out }' | $(MKHASH) md5
```

Comment:
```suggestion
find_md5=find $(wildcard $(1)) -type f $(patsubst -x,-and -not -path,$(DEP_FINDPARAMS) $(2)) -printf "%p%T@\n" | sort | sed 's#$(TOPDIR)##' | $(MKHASH) md5
```

Reply:
I used awk to handle situation with special char in TOPDIR like `/home/Ansuel#/`... 
this will result in 
sed `'s#/home/Ansuel#/##'` and sed will fail

Reply:
What would happen if they have `"` in the path?

Reply:
Mhh can a directory have `"` in the name? But again guess we need take a decision on this... Probably I'm too paranoic on what shittery an user may have in $TOPDIR

Reply:
In any case, we can also use " as separator for sed

Reply:
big idea! can we use a whitespace as separator? as we enforce no whitespace in the openwrt path?

Reply:
@aparcar should we use something like this?
`sed s$'\001'$TOPDIR$'\001'$'\001'`

Reply:
```
find $(pwd)/tools/flock/ -type f | sort | sed s$'\032'$TOPDIR$'\032'$'\032'
/tools/flock/Makefile
/tools/flock/src/flock.c
```

`\032` should be whitespace

Reply:
@jow- thoughts?

Reply:
@aparcar wait i notice the problem was another so i'm pushing a real fix... ignore this change since it's REALLY WRONG.

Reply:
(check the new commit find_md5 if you want to know the findings...)